tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleLivenessFailedTest.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
```

Comment:
2017-2019

Reply:
Since, this is a new file, shouldn't the year be just 2019?